notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -559,11 +575,16 @@ export default {
         });
       });
     },
-    displayMessage(message) {
+    displayMessage(message, keepAlert) {
       this.message=message.message;
       this.type=message.type;
       this.alert = true;
-      window.setTimeout(() => this.alert = false, 5000);
+      if (!keepAlert) {
+        window.setTimeout(() => this.alert = false, 5000);
+      }
+    },
+    onclose() {
+      localStorage.setItem(`displayAlert-${this.spaceDisplayName}`, 'already_display');
```

Comment:
What if the space's display name changes ? Better rename it according to an unchangeable attribute